pkg/ctl/enable/toolkit_test.go

```diff
@@ -0,0 +1,197 @@
+package enable
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	api "github.com/weaveworks/eksctl/pkg/apis/eksctl.io/v1alpha5"
+	. "github.com/weaveworks/eksctl/pkg/ctl/ctltest"
+)
+
+var _ = Describe("enable toolkit", func() {
+	var mockEnableToolkitCmd func(args ...string) *MockCmd
+
+	BeforeEach(func() {
+		mockEnableToolkitCmd = func(args ...string) *MockCmd {
+			return NewMockCmd(configureAndRun, "enable", args...)
+		}
+	})
+
+	When("--config-file is not provided", func() {
+		It("should fail", func() {
+			cmd := mockEnableToolkitCmd("toolkit")
+			out, err := cmd.Execute()
+			Expect(err).To(HaveOccurred())
+			Expect(err).To(MatchError("--config-file/-f <file> must be set"))
+			Expect(out).To(ContainSubstring("Usage"))
+		})
+	})
+
+	When("name arg is provided", func() {
+		It("should fail", func() {
+			cmd := mockEnableToolkitCmd("toolkit", "foo")
+			out, err := cmd.Execute()
+			Expect(err).To(HaveOccurred())
+			Expect(err).To(MatchError("name argument is not supported"))
+			Expect(out).To(ContainSubstring("Usage"))
+		})
+	})
+
+	When("--cluster is provided", func() {
+		It("should fail", func() {
+			cmd := mockEnableToolkitCmd("toolkit", "--cluster", "foo")
+			out, err := cmd.Execute()
+			Expect(err).To(HaveOccurred())
+			Expect(err).To(MatchError("unknown flag: --cluster"))
+			Expect(out).To(ContainSubstring("Usage"))
+		})
+	})
+
+	When("--region is provided", func() {
```

Comment:
These feel like brittle/superfluous tests since we're explicitly declaring the incompatible flags in `cmd/cmdutils/gitops.go` and they should ideally be handled through a single interface. Do  these CLI option tests provide any value, what do we think?

Reply:
huh actually I don't even need to mark them as incompatible, since I have removed those flags altogether.

so yes, these tests are superfluous, but for a different reason 😄